com.unity.netcode.gameobjects/Runtime/Serialization/Pooled/NetworkBufferPool.cs

```diff
@@ -12,8 +12,11 @@ public static class NetworkBufferPool
         private static Queue<WeakReference> s_OverflowBuffers = new Queue<WeakReference>();
         private static Queue<PooledNetworkBuffer> s_Buffers = new Queue<PooledNetworkBuffer>();
 
-        private const uint k_MaxBitPoolBuffers = 1024;
-        private const uint k_MaxCreatedDelta = 768;
+        // Increasing this value for various reasons:
+        // Some tests (i.e. MessageHandlerReceivedMessageServerClient) expect certain log messages and if we happen to hit the threshold it will cause those tests to fail
+        // This slows down tests having to log warnings repeatedly.
+        private const uint k_MaxBitPoolBuffers = 2048;
```

Comment:
This feels like it's just hiding a problem in our tests, or worse, in the framework... Why are we using more than a thousand buffers simultaneously?

Reply:
+1 :(

Reply:
Ok, I can put it back to 1024. Our unit tests will consume much more log file space and MessageHandlerReceivedMessageServerClient will fail because it is detecting the warning that we are exceeding 1024 buffers.

It is a terrible pooling system, I was just trying to provide plenty of room.  It isn't the amount we are allocating, it is the amount required before it spams a warning message.

Reply:
Just reverted the changes, this PR won't be able to pass Yamato tests.

Reply:
Well that's just crazy...it passed even though it fails on my end.
Go figure.,

Reply:
there is an old JIRA ticket here: MTT-656

Reply:
Yeah, increasing the k_MaxBitPoolBuffers only really prevents the warning message spam during unit tests which reduces processing overhead, time, and the size of the log files.  It doesn't pre-allocate buffers based on that size...just doesn't start warning until it reaches that threshold.   Keeping it at 1024 just means lots of console spam during unit and integration tests.